src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/TestForEmptyStringsUsingStringLengthTests.Fixer.cs

```diff
@@ -14,6 +14,42 @@ namespace Microsoft.NetCore.Analyzers.Runtime.UnitTests
     public class TestForEmptyStringsUsingStringLengthFixerTests
     {
         private const int c_StringLengthCodeActionIndex = 1;
+
+        [Fact]
+        public async Task CA1820_FixTestEmptyStringsUsingIsNullOrEmpty_WhenStringIsLiteral()
```

Comment:
Could you add a test with the other fixer too?

Reply:
Also, could you add 2 tests with the empty string on the left side? I am fine with adding the cases on the existing test(s).

Reply:
I'll add them soon.